types.go

```diff
@@ -40,6 +40,11 @@ type Account struct {
 	ImportedKeyCount int32
 }
 
+type AccountIterator struct {
```

Comment:
```suggestion
type AccountsIterator struct {
```
Also, the properties in this struct do not need to be public.